cg/models/cg_config.py

```diff
@@ -563,6 +581,31 @@ def mutacc_auto_api(self) -> MutaccAutoAPI:
             self.mutacc_auto_api_ = api
         return api
 
+    @property
+    def pacbio_post_processing_service(self) -> PacBioPostProcessingService:
+        service = self.__dict__.get("pacbio_post_processing_service_")
+        if service is None:
+            LOG.debug("Instantiating PacBio post-processing service")
+            run_data_generator = PacBioRunDataGenerator()
+            file_manager = PacBioRunFileManager()
+            metrics_parser = PacBioMetricsParser(file_manager=file_manager)
+            transfer_service = PacBioDataTransferService(metrics_service=metrics_parser)
+            store_service = PacBioStoreService(
+                store=self.status_db, data_transfer_service=transfer_service
+            )
+            hk_service = PacBioHousekeeperService(
+                hk_api=self.housekeeper_api,
+                file_manager=file_manager,
+                metrics_parser=metrics_parser,
+            )
+            service = PacBioPostProcessingService(
+                run_data_generator=run_data_generator,
+                hk_service=hk_service,
+                store_service=store_service,
+            )
+            self.pacbio_post_processing_service_ = service
+        return service
```

Comment:
Maybe we can group post processing services all together like.

cg_config.post_processing_services.pacbio / illumina / etc. ?

Reply:
fixed